pom.xml

```diff
@@ -68,6 +68,8 @@
         <geoserver.version>2.17.3</geoserver.version>
         <jts.version>1.17.0</jts.version> <!-- needs to track with gt/gs version -->
         <spatial4j.version>0.8</spatial4j.version> <!-- needs to track with jts version -->
+        <!-- match dependency version used by gt-referencing for convenience -->
```

Comment:
is org.ejml actually used by gt-referencing now? previously they were using the com.googlecode.efficient-java-matrix-library ejml library.

Reply:
Looks like: https://github.com/geotools/geotools/blob/main/pom.xml#L1085-L1089
https://github.com/geotools/geotools/blob/main/modules/library/referencing/pom.xml#L173-L177

Reply:
i wonder, do you know if they switched or if the old comment was wrong? good catch though

Reply:
Lemme see.... if I recall, GT was using vecmath and then switched to com.googlecode.efficient-java-matrix-library 0.26 and then upgrade to 0.34.  

The move from vecmath was motivated by licenses back in the day; I think I helped with that at FOSS4G NA 2015...  Can't remember when the 0.26 -> 0.34 transition happened.